x/gov/types/kumsg.go

```diff
@@ -97,6 +97,14 @@ func NewKuMsgDeposit(auth sdk.AccAddress, depositor AccountID, proposalID uint64
 	}
 }
 
+func (msg KuMsgDeposit) ValidateBasic() error {
+	msgData := MsgDeposit{}
```

Comment:
Should add ValidateBasic call for KuMsg

Reply:
Already add call of validaebasic for kumsg